docs/4_4_1_can.adoc

```diff
@@ -612,21 +611,19 @@ If a Network FMU does not support wake up, this operation can be ignored on the
-By using configuration parameters on Bus Terminal level, Network FMUs can be parameterized by the user or the Bus Simulation (by using e.g. the parameter propagation mechanism).
-This chapter specifies the configuration parameters that each CAN-specific Network FMU shall provide.
+This chapter defines parameters that Network FMU might provide to configure CAN-specific behavior.
 
 ====== Bus Notification Parameter [[low-cut-can-bus-notification-parameter]]
 For a detailed simulation, the CAN bus behavior regarding acknowledgment, bus errors and arbitration losses must be considered.
 A Bus Simulation can simulate these effects by sending bus notifications in terms of <<low-cut-can-confirm-operation, `Confirm-`>>, <<low-cut-can-bus-error-operation, `Bus Error-`>> and <<low-cut-can-arbitration-lost-operation, `Arbitration Lost`>> operations to the Network FMUs.
 
 However, in scenarios where Network FMUs are connected directly to each other, or where the Bus Simulation does not simulate such effects, it must be possible to configure the Network FMU such that it does not wait for any response after a <<low-cut-can-transmit-operation, `Transmit`>> operation.
-Therefore, the <<low-cut-can-bus-notification-parameter, `BusNotifications`>> configuration parameter is introduced for each CAN Bus Terminal.
-If a Network FMU is not supporting bus notifications at all, the configuration parameter shall be omitted.
-If a Network FMU supports bus notifications, the configuration parameter shall be defined with a default value set to `false` to allow the Network FMU to be run natively in each simulation scenario.
-If the value of the configuration parameter is set to `false`, the Network FMU must not wait for any response after a <<low-cut-can-transmit-operation, `Transmit`>> operation ("fire-and-forget").
-If the Network FMU shall be configured to handle responses in the form of <<low-cut-can-confirm-operation, `Confirm-`>>, <<low-cut-can-bus-error-operation, `Bus Error-`>> and <<low-cut-can-arbitration-lost-operation, `Arbitration Lost`>> operations, the <<low-cut-can-bus-notification-parameter, `BusNotifications`>> configuration parameter shall be set to `true`.
-A Network FMU must always be able to handle the case that the Bus Simulation does not provide <<low-cut-can-confirm-operation, `Confirm-`>>, <<low-cut-can-bus-error-operation, `Bus Error-`>> or <<low-cut-can-arbitration-lost-operation, `Arbitration Lost`>> operations.
-Since a Bus Simulation is allowed to provide bus notification regardless of the <<low-cut-can-bus-notification-parameter, `BusNotifications`>> configuration parameter, Network FMUs must silently ignore such notifications if the value of <<low-cut-can-bus-notification-parameter, `BusNotifications`>> is set to `false` or the parameter is omitted. +
+Therefore, a parameter with memberName=<<low-cut-can-bus-notification-parameter, `BusNotifications`>> can be added to a CAN Bus Terminal. +
```

Comment:
To the "sub" config terminal .... thats why I called it "configuration parameter"

Reply:
see above